.github/CODEOWNERS

```diff
@@ -72,6 +74,7 @@ processor/resourcedetectionprocessor/                @open-telemetry/collector-c
 processor/resourcedetectionprocessor/internal/azure  @open-telemetry/collector-contrib-approvers @mx-psi
 processor/routingprocessor/                          @open-telemetry/collector-contrib-approvers @jpkrohling
 processor/spanmetricsprocessor/                      @open-telemetry/collector-contrib-approvers @albertteoh
+processor/spanprocessor/                              @open-telemetry/collector-contrib-approvers @boostchicken
```

Comment:
I feel bad for nit-picking this, but there's an extra space here :-)

Reply:
DOnt feel bad, details matter

Reply:
Fixed!